osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -179,16 +173,21 @@ private void load()
             {
                 Text = SkinSettingsStrings.ExportSkinButton;
                 Action = export;
+            }
+
+            protected override void LoadComplete()
+            {
+                base.LoadComplete();
 
                 currentSkin = skins.CurrentSkin.GetBoundCopy();
-                currentSkin.BindValueChanged(skin => Enabled.Value = skin.NewValue.SkinInfo.ID > 0, true);
+                currentSkin.BindValueChanged(skin => Enabled.Value = skin.NewValue.SkinInfo.PerformRead(s => s.IsManaged), true);
```

Comment:
I believe this should be
```suggestion
                currentSkin.BindValueChanged(skin => Enabled.Value = skin.NewValue.SkinInfo.PerformRead(s => !s.IsProtected), true);
```
because in realm default skins *are* managed, and as such it is possible to export the default skins and get an empty archive back.